composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "behat/behat": "^3.6.1",
-        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.3 || ^6.0",
-        "symfony/proxy-manager-bridge": "^4.4 || ^5.3 || ^6.0"
+        "php": "^8.1",
+        "behat/behat": "dev-master",
```

Comment:
this should stay as `^3.6.1` IMO. Your CI job for Symfony 7 should be configuring `minimum-stability: dev` for now until the support is part of a Behat release.